spec/index.bs

```diff
@@ -3136,50 +3174,56 @@ The {{GPUMapMode}} flags determine how a {{GPUBuffer}} is mapped when calling
         GPU again.
 
         <div algorithm=GPUBuffer.unmap>
-            **Called on:** {{GPUBuffer}} |this|.
-
-            **Returns:** {{undefined}}
-
-            1. If any of the following requirements are unmet, [$generate a validation error$] and stop.
+            <div data-timeline=content>
+                **Called on:** {{GPUBuffer}} |this|.
 
-                <div class=validusage>
-                    - |this|.{{GPUBuffer/[[state]]}} must be [=buffer state/mapped at creation=],
-                        [=buffer state/mapping pending=], or [=buffer state/mapped=].
+                **Returns:** {{undefined}}
 
-                    Note: It is valid to unmap an [=invalid=] {{GPUBuffer}} that is
-                        [=buffer state/mapped at creation=] because the [=Content timeline=]
-                        might not know it is an error {{GPUBuffer}}. This allows the temporary
-                        {{GPUBuffer/[[mapping]]}} memory to be freed.
-                </div>
+                1. For each promise |p| in |this|.{{GPUBuffer/[[map_promises]]}}:
 
-            1. If |this|.{{GPUBuffer/[[state]]}} is [=buffer state/mapping pending=]:
+                    1. Reject |p| with an {{AbortError}}.
```

Comment:
OK, this isn't quite what Chromium currently does. I need to figure out if I can easily encode that behavior here, I guess. https://github.com/gpuweb/gpuweb/issues/3271#issuecomment-1210711012